api/internal/handler/route/route.go

```diff
@@ -175,7 +175,7 @@ func generateLuaCode(script map[string]interface{}) (string, error) {
 	}
 
 	cmd := exec.Command("sh", "-c",
-		"cd "+conf.DagLibPath+" && lua cli.lua "+
+		"cd "+conf.WorkDir+"/dag-to-lua && lua cli.lua "+
```

Comment:
seems wrong.

we should use `conf.dag_lib_path`, all right?

Reply:
I'will remove that config.users should not need to this dir.